Documentation/Maintenance/Release.md

```diff
@@ -333,12 +333,15 @@ Update Remote Modules
 In order to have the latest versions for all remote modules, and have them use
 the latest ITK tag, the following steps should be performed:
 
-1. Update the ITK tag used in the `azure-pipelines.yml` CI configuration and the
-`setup.py` Python setup files using the [](https://github.com/InsightSoftwareConsortium/ITK/tree/master/Utilities/Maintenance/)
+1. Update the ITK tag used in the `azure-pipelines.yml` CI configuration and
+the `setup.py` Python setup files, and update the remote module Python package
+version to a new major version using the [UpdateRequiredITKVersionInFiles.sh](https://github.com/InsightSoftwareConsortium/ITK/tree/master/Utilities/Maintenance/UpdateRequiredITKVersionInFiles)
```

Comment:
To be more specific, can we rename *UpdateRequiredITKVersionInFiles.sh* to *UpdateRequiredITKVersionInRemoteModule.sh*? This will help folks look at the script name understand its purpose.

Reply:
Looks reasonable: #664.

I will rebase this PR and do the corresponding changes once the above gets merged.